src/Angular.js

```diff
@@ -1602,6 +1622,11 @@ function angularInit(element, bootstrap) {
     }
   });
   if (appElement) {
+    if (!isAutoBootstrapAllowed) {
```

Comment:
Yeah, I thought it might be intentional. I don't expect anyone to be loading angular over `resource:` on non-extension pages, but you never know :smiley:

Hm...so, if it is a supported usecase (loading angular.js over `resource:` on non-extension pages), isn't this chage breaking? In that case, we should have a breaking change notice.